Cargo.toml

```diff
@@ -36,10 +36,10 @@ tokio = { version = "1.35.1", features = ["rt-multi-thread", "signal"] }
 tracing = "0.1.40"
 tracing-subscriber = { version = "0.3.18", features = ["env-filter"] }
 metrics = "0.22.0"
-datafusion = "37.0.0"
+datafusion = "37.1.0"
 arrow = "51.0.0"
 arrow-flight = "51.0.0"
-duckdb = { git = "https://github.com/spiceai/duckdb-rs.git", rev = "a0ae8ed71bef6dff779f5ce0a37038b0b916e0ca" }
+duckdb = { git = "https://github.com/duckdb/duckdb-rs.git", rev = "a1aa55aff22b75e149e9cf7cface6464b3dc0ccc" }
```

Comment:
Is there specific reason we don't just use version from rust crates? 
[v0.10.2](https://crates.io/crates/duckdb/0.10.2)

Reply:
Ah, good catch. I didn't know it was published there yet. Will update.

Reply:
they JUST released it a few hours ago :).